docker/worker/Dockerfile

```diff
@@ -13,9 +13,9 @@ RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
 
 # set-up qemu
 RUN mkdir -p /root/qemu
-ADD kvm-mknod.sh /qemu/kvm-mknod.sh
+ADD kvm-mknod.sh /root/qemu/kvm-mknod.sh
 ADD run_openqa_worker.sh /run_openqa_worker.sh
-RUN chmod +x /root/qemu/*.sh && /root/qemu/kvm-mknod.sh && \
+RUN chmod +x /root/qemu/kvm-mknod.sh && /root/qemu/kvm-mknod.sh && \
     chmod a+x /run_openqa_worker.sh && \
```

Comment:
I wondered why we should even need these chmod calls. I created https://github.com/os-autoinst/openQA/pull/3547 to simplify, if you agree. This just makes the scripts executable and copy as-is into the container build environment.